ckanext/canada/plugins.py

```diff
@@ -789,6 +793,21 @@ def datastore_delete(up_func, context, data_dict):
     return result
 
 
+class CanadaActivity(p.SingletonPlugin):
+    p.implements(p.IActions)
+    p.implements(p.IConfigurer)
+
+    def get_actions(self):
+        return ({'datastore_upsert':datastore_upsert,
+                'datastore_delete': datastore_delete})
+
+    def update_config(self, config):
+        logic_validators.object_id_validators.update({
+            'changed datastore': logic_validators.package_id_exists,
+            'deleted datastore': logic_validators.package_id_exists,
+        })
```

Comment:
While we're cleaning this up, can this be merged with one of our other plugins? I don't think we ever want this disabled on one of our sites, right?

Reply:
@wardi okay `canada_activity` is removed and it's plugin code has been migrated into `canada_public`: https://github.com/open-data/ckanext-canada/pull/1313/commits/9d44b352ac70fff6f8345d43dccf2f9683879365